api/swagger.yaml

```diff
@@ -8331,6 +8331,15 @@ paths:
           description: "BuildKit output configuration"
           type: "string"
           default: ""
+        - name: "version"
+          in: "query"
+          type: "string"
+          default: "1"
+          description: |
+            Version of the builder backend to use.
+
+            - `1` is the first generation classic builder in the Docker daemon (default)
+            - `2` is [BuildKit](https://github.com/moby/buildkit)
```

Comment:
Does Swagger / OpenAPI v2 allow enums for query parameters? Looks like it does; perhaps we should use that


```yaml
        - name: "foo"
          in: "query"
          description: "BuildKit output configuration"
          type: "string"
          enum:
            - "hello"
            - "world"
          default: "hello"
```


<img width="881" alt="Screenshot 2024-02-01 at 15 56 31" src="https://github.com/moby/moby/assets/1804568/04d610b5-51f3-41bd-9bdf-087fe660a576">

Reply:
Oh thanks, done!